buildSrc/src/main/kotlin/BuildExtensions.kt

```diff
@@ -228,6 +228,22 @@ fun Project.configureTaskDependencies() {
 val Project.productionModules: Iterable<Project>
     get() = rootProject.subprojects.filter { !it.name.contains("-tests") }
 
+/**
+ * Obtains the names of the [productionModules].
```

Comment:
Please adjust docs to `val Project.productionModules`, I see in preview that it also `Obtains all modules names ...` while the names are obtained exactly by this extension.